### DISCUSSION TITLE
More granular/persistent permission management

### DISCUSSION THREAD

**Discussion by ceuk** [Jelmerro/Vieb#551] [Q&A]
For permissions set to `ask` is there a way to either persist allowing permission beyond the session (i.e. when I click allow for a site, it will *always* be allowed for that site) -- or -- is there a more granular way to allow permissions on a site-by-site basis instead of just a blanket `allow` in the settings?

**Comment by Jelmerro** [comment]
You can use the settings `permissionsallowed` and related settings like `permissionsasked` and `permissionsblocked` for that. Please see `:h permissionsallowed` for details. I will try to answer questions if you have them, but the help should be complete in this regard.
